packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -105,7 +105,13 @@ variable "setup_raid" {
 }
 
 variable "reservation_ids" {
-  description = "Specify Packet hardware_reservation_id for instances. A map where the key format is 'worker-${index}' and the associated value is the reservation id string. Nodes not present in the map will use no reservation id. Example: reservation_ids = { worker-0 = '<reservation_id>' }"
+  description = "Specify Packet hardware_reservation_id for instances. A map where the key format is 'worker-${index}' and the associated value is the reservation id string. Nodes not present in the map will use the value of reservation_ids_default var. Example: reservation_ids = { worker-0 = '<reservation_id>' }"
```

Comment:
>Nodes not present in the map will use the value of reservation_ids_default var.

I suggest the following phrasing:

>Nodes not present in the map will use the value of the `reservation_ids_default` variable.

Reply:
Thanks, changed it locally will add a new PR with these :)